### ISSUE TITLE
Unsupported Erlang version (erl 20)

### ISSUE THREAD

**Issue by khodzha** [goertzenator/erlang_nif-sys#27]
i've got this error using rustler crate:
```
--- stdout
Unsupported Erlang version.
```

my erlang version:
```
Erlang/OTP 20 [RELEASE CANDIDATE 2] [erts-9.0] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false]

Eshell V9.0  (abort with ^G)
```

erlang_nif-sys version from Cargo.lock 
```
[[package]]
name = "erlang_nif-sys"
version = "0.6.1"
```

**Comment by goertzenator** [comment]
I do have an otp20 branch in progress.  I also know that the NIF API is going to change before the final otp-20 release lands, so the intention is to not release a new crate until otp-20 is formally released.

**Comment by khodzha** [comment]
thank you for clarifying this

i will mark this issue as resolved then

**Comment by goertzenator** [comment]
Reopening to track progress of otp20 branch.

otp20 branch is committed.  It needs `enif_select` testing and backwards compatibility work, but it should be suitable if you are keen on using otp20 with Rust right now.

**Comment by cgregfreeman** [comment]
Thank you! My apologies for the duplicate issue

**Comment by nitingupta910** [comment]
Getting same error with erlang 20.2

**Comment by goertzenator** [comment]
I plan on doing some maintenance on this project during my Christmas holiday.

**Comment by goertzenator** [comment]
Fixed in 447e4ac70cfe1becc20a7bd02d4c21b0cfc22763.

I'm having some issues with crates.io.  Will publish as soon as I can.

**Comment by nitingupta910** [comment]
Thanks!
